packages/webdriver/src/types.ts

```diff
@@ -163,12 +163,13 @@ export interface FirefoxLogObject {
 }
 
 export interface FirefoxOptions {
-    binary?: string,
-    args?: string[],
-    profile?: string,
-    log?: FirefoxLogObject,
+    debuggerAddress: string
+    binary?: string
+    args?: string[]
+    profile?: string
+    log?: FirefoxLogObject
```

Comment:
why drop the commas? does the linker expect one way or the other?

Reply:
I don't think there is any separator needed when you define an interface. Semicolons are optional and commas will be ignored.